lib/std/os/linux/x86.zig

```diff
@@ -149,8 +149,11 @@ pub fn clone() callconv(.Naked) usize {
         \\  int $128
         \\  testl %%eax,%%eax
         \\  jnz 1f
```

Comment:
Needs to be inverted just like `x86_64` so that the parent branch doesn't appear at an address greater than `.cfi_undefined`.

Reply:
Good catch, and seems like the same should be done for `powerpc` and `sparc64` as well.